Description: Fix build failure because of removal of ARCHIVE_LIBRARY_VERSION.
 Use ARCHIVE_VERSION_STRING instead.
Author: Andres Mejia <amejia@debian.org>

--- a/libarchive/libarchive.c
+++ b/libarchive/libarchive.c
@@ -141,7 +141,7 @@
 
   module_info->ID = g_strdup ("libarchive_plugin");
   module_info->Name = g_strdup ("libarchive plugin");
-  module_info->About = g_strdup_printf ("version %s, build date: %s\nusing %s\n", VERSION, BUILD_DATE, ARCHIVE_LIBRARY_VERSION);
+  module_info->About = g_strdup_printf ("version %s, build date: %s\nusing %s\n", VERSION, BUILD_DATE, ARCHIVE_VERSION_STRING);
   module_info->Copyright = g_strdup ("Plugin Copyright (C) 2008-2009 Tomáš Bžatek\nlibarchive sources Copyright (c) 2003-2007 Tim Kientzle");
 
   return module_info;
